### DISCUSSION TITLE
Library layout

### DISCUSSION THREAD

**Discussion by Editpo** [rern/rAudio#1746] [Fixed Issues]
20240818
The letters/menu down the right hand side now overlaps the coverart. On iPad
![IMG_0532](https://github.com/user-attachments/assets/23fa11f1-27d3-4802-b3c9-8f9c5395b41d)

**Comment by AndyFx12** [comment]
Same here - Firefox 129 on Linux.

**Reply by rern** [reply]
Please try the UPDATE.

**Comment by AndyFx12** [comment]
Works fine, now.
Thanks!

**Comment by Editpo** [comment]
Not seeing update yet

**Comment by hjfisch** [comment]
Long press update button .
Seems to be fixed .

**Comment by Editpo** [comment]
Thanks.
After update I’m not getting the library show up! Tried everything I can think of.
I’ll redo the sd card later

**Comment by Sagnasty** [comment]
@Editpo - take a look at #1748 

Appears to be a common issue that was recently introduced .
